x-pack/test/detection_engine_api_integration/security_and_spaces/group10/read_privileges.ts

```diff
@@ -478,31 +478,31 @@ export default ({ getService }: FtrProviderContext) => {
         username: 'detections_admin',
         has_all_requested: false,
```

Comment:
I've noticed `detections_admin` had different privileges in ESS and Serverless in my [PR](https://github.com/elastic/kibana/pull/170131). I guess your change here will fail in ESS.